docs/src/testing/contract-collection/new-mechanism.md

```diff
@@ -0,0 +1,36 @@
+# How Contracts Are Collected
+
+For the [`declare`](../../appendix/snforge-library/declare.md) to function, snforge must collect and call build on
+contracts in the package. By default, if using Scarb version greater or equal to 2.8.3, snforge will combine test
+collection and contract collection steps.
```

Comment:
Not sure if it should be:
- ... function to work, snforge must collect...
- ... function, snforge must collect...

```suggestion
For the [`declare`](../../appendix/snforge-library/declare.md) function to, snforge must collect and call build on
contracts in the package. By default, if using Scarb >= 2.8.3, snforge will combine test
collection and contract collection steps.
```

Reply:
I meant "declare function", not "for declare to operate (function)".

Reply:
For the declare to function, ...

Reply:
But changed to `>=`.